src/PrestaShopBundle/Controller/Admin/Configure/AdvancedParameters/ProfileController.php

```diff
@@ -264,6 +265,13 @@ public function bulkDeleteAction(Request $request)
     protected function getErrorMessages()
     {
         return [
+            ProfileConstraintException::class => [
+                ProfileConstraintException::INVALID_NAME => $this->trans(
+                    'Your entry in field name exceeds max length %d chars (incl. HTML tags)',
```

Comment:
It works for both first and last name fields, right? In this case, I would write _Your entry in the %s field cannot be longer than %limit% characters (incl. HTML tags)_, localized in the same domain.

Reply:
It's not a person name, it's a name we give to a profile, like `SuperAdmin`, `Logistician`, `Salesman`, etc... But if you still think I should change it I'll do it! Wdyt?

Reply:
My bad, I was on the employee tab... What about _This field cannot be longer than %limit% characters (incl. HTML tags)_ in Admin.Notifications.Error?